<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="inputLayout" stretch="0,0,0,0,0,0,1,0">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="programNameLabel">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>RANBAB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="groupCountLabel">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>그룹 수</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="groupCountLineEdit">
         <property name="maximumSize">
          <size>
           <width>196</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMask">
          <string>00</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseFileButton">
         <property name="maximumSize">
          <size>
           <width>196</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>파일 선택</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="groupEntryText">
         <property name="maximumSize">
          <size>
           <width>196</width>
           <height>199</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPeopleCountLabel">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>총 인원 : 0명</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="catImageLabel">
         <property name="font">
          <font>
           <family>나눔고딕</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateGroupButton">
         <property name="minimumSize">
          <size>
           <width>196</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>196</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>생성하기</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="outputLayout" stretch="0,0,1,0">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>랜덤 조 편성</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
         <item>
          <widget class="QLabel" name="resultLabel">
           <property name="font">
            <font>
             <family>나눔고딕</family>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>결과</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyButton">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>나눔고딕</family>
            </font>
           </property>
           <property name="text">
            <string>복사</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="resultWidget" native="true">
         <property name="font">
          <font>
           <family>나눔고딕</family>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>542</width>
               <height>443</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::Shape::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Shadow::Plain</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QPushButton" name="saveButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>나눔고딕</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>저장</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
